key/src/address.rs

```diff
@@ -36,6 +36,19 @@ pub struct FullAddress {
 }
 
 impl FullAddress {
+    pub fn create_version0(network_id: u64, address: Address) -> Result<Self, Error> {
```

Comment:
Why don't you pass `0` as an argument?

Reply:
The arguments can be different for the different version, in my expectation.